bin/views/highlight.html

```diff
@@ -7,5 +7,24 @@
     </head>
     <body>
 {{!codehl}}
+        <div class="controls">
+            % if parentid:
+            <a class="control" title="Voir la révision précédente" href="/{{parentid}}.{{ext}}">
+                <!-- missing icon -->
```

Comment:
Que fait-on pour cette icône manquante ?

Reply:
C'est clairement un "Copier" comme tu l'as dit, ailleurs, je ne me vois pas l'utiliser pour voir la révision d'avant.
> Je vois plus ca comme un "Copier", en tout cas ca fait très bizarre de se dire que ca représente un "Modifier"

Reply:
Quelque chose comme ça peut être ? 

![Frame 1(1)](https://user-images.githubusercontent.com/62183512/106359966-41d76e80-6316-11eb-8c0a-8161008d0a21.png)

```suggestion
                <svg viewBox="0 0 20 20" xmlns="http://www.w3.org/2000/svg" fill="currentColor">
                    <path fill-rule="evenodd" clip-rule="evenodd" d="M4.58579 2.58579C4.96086 2.21071 5.46957 2 6 2H10.586C11.1164 2.00011 11.625 2.2109 12 2.586L15.414 6C15.7891 6.37499 15.9999 6.88361 16 7.414V16C16 16.5304 15.7893 17.0391 15.4142 17.4142C15.0391 17.7893 14.5304 18 14 18H6C5.46957 18 4.96086 17.7893 4.58579 17.4142C4.21071 17.0391 4 16.5304 4 16V4C4 3.46957 4.21071 2.96086 4.58579 2.58579ZM9.85571 7.29381C9.71506 7.15316 9.5243 7.07414 9.32538 7.07414C9.12647 7.07414 8.93571 7.15316 8.79505 7.29381C8.6544 7.43447 8.57538 7.62523 8.57538 7.82414V11.3241C8.57542 11.4739 8.6203 11.6203 8.70424 11.7443C8.78819 11.8684 8.90735 11.9644 9.04638 12.0201L11.5464 13.0201C11.7295 13.087 11.9315 13.08 12.1095 13.0005C12.2875 12.921 12.4276 12.7753 12.5 12.5943C12.5724 12.4133 12.5715 12.2112 12.4975 12.0308C12.4235 11.8505 12.2821 11.706 12.1034 11.6281L10.0754 10.8161V7.82414C10.0754 7.62523 9.99637 7.43447 9.85571 7.29381Z"/>
                </svg>
```

Reply:
Ca a l'air magnifique, mais a mon avis, faudrait juste que "les flèches" soient plus "large", parce que ca fait un peu fin.
![image](https://user-images.githubusercontent.com/48163546/106360409-92e86200-6318-11eb-9c3f-3884929026e1.png)
Un peu comme la largeur du "plus", mais moins

Reply:
Quelque chose comme ça :

![Frame 4(1)](https://user-images.githubusercontent.com/62183512/106361118-8239eb00-631c-11eb-8008-eb49454ed9f3.png)


```suggestion
                <svg viewBox="0 0 20 20" xmlns="http://www.w3.org/2000/svg" fill="currentColor"><path d="M4.6 2.6C5 2.2 5.5 2 6 2H10.6C11.1 2 11.6 2.2 12 2.6L15.4 6C15.8 6.4 16 6.9 16 7.4V16C16 16.5 15.8 17 15.4 17.4 15 17.8 14.5 18 14 18H6C5.5 18 5 17.8 4.6 17.4 4.2 17 4 16.5 4 16V4C4 3.5 4.2 3 4.6 2.6ZM9.7 7.3C9.5 7.1 9.3 7 9 7 8.7 7 8.5 7.1 8.3 7.3 8.1 7.5 8 7.7 8 8V12.7C8 12.9 8.1 13.1 8.2 13.2 8.3 13.4 8.4 13.5 8.6 13.6L12 14.9C12.2 15 12.5 15 12.7 14.9 12.9 14.8 13.1 14.6 13.2 14.4 13.3 14.1 13.3 13.8 13.2 13.6 13.1 13.4 12.9 13.2 12.7 13.1L10 12V8C10 7.7 9.9 7.5 9.7 7.3Z"/></svg>
```